alpha/apps/kaltura/lib/myXmlUtils.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+class myXmlUtils
+{
+	public static function validateXmlFileContent($filePath)
+	{
+		if(!$filePath)
+		{
+			return true;
+		}
+
+		$fileType = kFileUtils::getMimeType($filePath);
+		if(strpos($fileType, 'html') !== false || strpos($fileType, 'xml') !== false)
+		{
+			$xmlContent = kFile::getFileContent($filePath);
+			if($xmlContent)

```

Comment:
No need to check xmlContent, since if file was empty, we would not found mimeType xml or html